src/Workspaces/Remote/Core/Microsoft.CodeAnalysis.Remote.Workspaces.csproj

```diff
@@ -48,6 +48,6 @@
     <InternalsVisibleTo Include="Roslyn.Services.Test.Utilities" />
     <InternalsVisibleTo Include="Microsoft.CodeAnalysis.EditorFeatures.Test.Utilities" />
     <InternalsVisibleTo Include="Roslyn.Services.UnitTests.Utilities" />
-    <InternalsVisibleTo Include="Microsoft.VisualStudio.Completion.Tests" Key="$(CompletionTestsKey)" />
+    <InternalsVisibleTo Include="Microsoft.VisualStudio.Completion.Tests" Key="$(CompletionTestsKey)" WorkItem="https://github.com/dotnet/roslyn/issues/35081" />
```

Comment:
Should tests be marked as "doesn't run in Visual Studio?" Although I'm kinda horrified there's IVT in general so all for getting rid of it, just asking.

Reply:
Tests do run in Visual Studio, at least in the sense that an API change that breaks a test assembly will block our insertions.